r/src/compute.cpp

```diff
@@ -180,6 +180,15 @@ std::shared_ptr<arrow::compute::FunctionOptions> make_compute_options(
     return out;
   }
 
+  if (func_name == "element_wise_min" || func_name == "element_wise_max") {
+    using Options = arrow::compute::ElementWiseAggregateOptions;
+    bool skip_nulls = false;
```

Comment:
As mentioned here... https://github.com/apache/arrow/blob/57ecc73e6153fea04e0ac0d13792ba0abb0dd779/cpp/src/arrow/compute/kernels/scalar_compare.cc#L472... `skip_nulls = true` is the default, so let's make it the default here too:
```suggestion
    bool skip_nulls = true;
```

Reply:
The default of `pmin()` and `pmax()` is `na.rm = FALSE`, and the definitions you added in `dplyr-functions.R` are consistent with that 👍 so that's good. Generally here in `compute.cpp`, the options defaults should be consistent with the defaults of the C++ function options.

Reply:
thanks, yes i went full R to the bottom fro the smae reason